### ISSUE TITLE
SweeterColor darker than actual color

### ISSUE THREAD

**Issue by droidery** [jathu/sweetercolor#7]
The color from SweeterColor doesn't match the hex color. Here's a screenshot comparing the colors using `#EB6E2F` set with XCode's storyboard, Inkscape, and SweeterColor:

<img width="748" alt="sweet" src="https://cloud.githubusercontent.com/assets/25188027/22636624/c3bbfe6c-ec90-11e6-9f34-f19b84ac0237.png">

NB: The screenshot above has colors more vibrant than they appear, I assume due to image compression from GitHub or the macOS screenshot util.

To reproduce:
1. Create one `UIView` in Storyboard, set its color to #EB6E2F
2. Create another `UIView` in code set the background color. eg `sweeterColorView.backgroundColor = UIColor(hex: "#EB6E2F")`
3. Open Adobe PS or Inkscape, set the color to #EB6E2F
4. Use the macOS Digital Color Meter again, the color from XCode and Adobe PS is correct, but SweeterColor has the wrong color

**Comment by jathu** [comment]
I'm not really sure what the problem is. However, when I tried initializing the colors in three different ways, I get the same results. 

![1](https://cloud.githubusercontent.com/assets/4391247/25753161/83fee3ec-3188-11e7-8cbe-dfe506196a86.png)

As you can see, this color is different from the one you will get in photoshop. The result here has an RGB of (232, 98, 42). Where as `#EB6E2F` is really (235, 110, 47). Even the native UIColor initializer fails.

**Comment by droidery** [comment]
The issue I was getting was a mismatch between the color I added in Storyboard, and the color I added programmatically with SweeterColor, not sure if these helps.

On an actual device, it can be quite noticeable.

**Comment by droidery** [comment]
@jathu I came across this issue about different color spaces.

https://stackoverflow.com/questions/27628473/uinavigationbar-background-color-setted-from-storyboard-differ-from-the-color-se

Do you think this could be related?

**Comment by cruisediary** [comment]
Yes I think is related to @ghost 's link Xcode storyboard support various RGB color options.

In my Mac actually Sketch color picker find Author's `SweeterColor`color is `#EB632F` correctly but `Inkscape` and `Xcode` related author's option is `#FC6509`
